src/main/res/layout/main.xml

```diff
@@ -8,14 +8,20 @@
     >
     <!-- The main content view -->
     <LinearLayout
-        xmlns:android="http://schemas.android.com/apk/res/android"
         android:id="@+id/home_root"
         android:layout_width="fill_parent"
         android:layout_height="fill_parent"
         android:divider="?android:attr/dividerVertical"
         android:orientation="vertical"
         android:showDividers="middle"
         >
+        <android.support.v7.widget.Toolbar
+            android:id="@+id/main_actionbar"
+            android:layout_height="wrap_content"
+            android:layout_width="match_parent"
+            android:minHeight="?attr/actionBarSize"
+            android:background="?attr/colorPrimary"
+            />
```

Comment:
For **this** set of changes I didn't need to use a toolbar as the actionbar in the main view, but we will later so might as well do it now.